docs/README.md

```diff
@@ -332,3 +332,4 @@ sections to be referred from elsewhere (e.g. `notify` a disable user is allowed
 | Parameter | Required/Optional | Data Type | Description                                            |
 | --------- | ----------------- | --------- | ------------------------------------------------------ |
 | `enabled` | Optional          | Boolean   | Disable corresponding configuration if set to `false`. |
+](https://github.dev/spawnpoint/ai-marketplace-monitor/blob/category-freestuff-fix/docs/README.md)
```

Comment:
Not sure what you try to add here.

By documentation, I simply means adding "free" to "category can be vehicles, proper ..."